pinc/stages.inc

```diff
@@ -261,110 +45,337 @@ $ELR_round = get_Round_for_round_number(1);
+            130000 => _('Harmoniser of Texts'),
+            140000 => _('Formatting Architect'),
+            150000 => _('Preserver of Texts'),
+        ]
+    );
+}
+
+function define_other_activities()
+{
+    global $code_url, $post_processing_forum_idx, $post_processing_forum_idx;
+
+    new Pool(
+        'PP',
+        _('Post-Processing'),
+        ['F1' => 400],
+        'REQ-AUTO',
+        '',
+        null, // access_change_callback
+        _('After going through various rounds of proofreading and formatting, the books need to be massaged into a final e-text.'),
+        'post_proof.php',
+        PROJ_POST_FIRST_CHECKED_OUT,
+        PROJ_POST_FIRST_AVAILABLE,
+        _("Manager"),
+        'username',
+        [
+            "<p>",
+            "<b>" . _("First Time Here?") . "</b>",
+            sprintf(
+                _("Please read the <a href='%s'>Post-Processing FAQ</a> as it covers all the steps needed to post-process an e-text."),
+                "$code_url/faq/post_proof.php"
+            ),
+            _("Select an easy work to get started on (usually fiction with a low page count is a good starter book; projects whose manager is BEGIN make excellent first projects for a new post-processor)."),
+            sprintf(_("Check out the <a href='%s'>Post-Processing Forum</a> to post all your questions."), get_url_to_view_forum($post_processing_forum_idx)),
+            _("If nothing interests you right now, check back later and there will be more!"),
+            "</p>",
+
+            "<p>",
+            _("Each book listed below has gone through multiple rounds of proofreading and formatting, and now needs to be massaged into a final e-text."),
+            _("Once you have checked out and downloaded a book it will remain checked out to you until you check it back in."),
+            _("When you have finished your work on the book, select <i>Upload for Verification</i> from the drop-down list for that project."),
+            _("If you have several files to submit for a single project (say a text and HTML version), zip them up together first."),
+            "</p>",
+        ]
+    );
+
+    // -----------------------------------------------------------------------------
+
+    new Stage(
+        'SR',
+        _('Smooth Reading'),
+        [],
+        'IMMEDIATE',
+        '',
+        null, // access_change_callback
+        _('Before a final e-text is posted to PG, it can be optionally uploaded for Smooth Reading. Anyone can volunteer to Smooth Read a text, which involves reading through the text for smoothness, marking possible errors and returning it to the PPer.'),
+        null,
+        "tools/post_proofers/smooth_reading.php"
+    );
+
+    // -----------------------------------------------------------------------------
+
+    new Pool(
+        'PPV',
+        _('Post-Processing Verification'),
+        [],
+        'NOREQ', // "Peer approval. Also gives F2 access."
```

Comment:
Is the comment true? PPV gives F2 access? I didn't know that. Or maybe I've misread it.

Reply:
No, PPV no longer gives F2 access. It used to until somewhere around 2007-2009, but apparently the comment was never revised. By the time I came along in 2009, it was no longer true.